Copy edit FAQs (#767)
timescale/docs: timescaledb/overview/faq/faq-postgres.md

To summarize, TimescaleDB offers:
* Ease-of-use: TimescaleDB is far easier to use because creating partitions (or what we call
"chunks") is automatically performed for the user. All of the complexity of automatic
partitioning is abstracted away behind a "hypertable," which users interact with just as
they would with a PostgreSQL table.
* Much higher ingest scale: TimescaleDB sees throughput more than 20X that of

> PostgreSQL once tables reach moderate size (for example, 10s of millions of rows).

Author: ```suggestion
PostgreSQL once tables reach moderate size (for example, 10&nbsp;s of millions of rows).
```

> PostgreSQL once tables reach moderate size (for example, 10s of millions of rows).

Author: ```suggestion
<!-- vale Google.Units = NO -->
PostgreSQL once tables reach moderate size (for example, 10s of millions of rows).
<!-- vale Google.Units = YES -->
```

To summarize, TimescaleDB offers:
* Ease-of-use: TimescaleDB is far easier to use because creating partitions (or what we call
"chunks") is automatically performed for the user. All of the complexity of automatic

> partitioning is abstracted away behind a "hypertable," which users interact with just as

Reviewer: ```suggestion
partitioning is abstracted away behind a hypertable, which users interact with just as
```

they would with a PostgreSQL table.

> * Much higher ingest scale: TimescaleDB sees throughput more than 20X that of 

Reviewer: ```suggestion
* Much higher ingest scale: TimescaleDB sees throughput more than 20 times that of 
```

<!-- vale Google.Units = NO -->

> PostgreSQL once tables reach moderate size (for example, 10s of millions of

Reviewer: ```suggestion
PostgreSQL once tables reach moderate size (for example, tens of millions of
```
